src/Filesystem/F.php

```diff
@@ -783,6 +783,39 @@ public static function size(string|array $file): int
 		}
 	}
 
+	/**
+	 * Continously outputs the file between the provided range
+	 */
+	public static function stream(
+		string $file,
+		int $start = 0,
+		int $end = null
+	): void {
+		$size   = static::size($file);
+		$end  ??= $size;
+
```

Comment:
I think there should be checks for `start < 0`, `start >= end` and `end > size` here as well, throwing an exception to provide extra safety and for those who use this method directly.